sdk/include/reactos/shellutils.h

```diff
@@ -597,21 +597,14 @@ void DumpIdList(LPCITEMIDLIST pcidl)
     DbgPrint("End IDList Dump.\n");
 }
 
-struct CCoInit
+template<HRESULT(WINAPI*InitFunc)(void*), void(WINAPI*UninitFunc)()> struct CCoInitBase
```

Comment:
Some spacing to ease reading.
```suggestion
template <HRESULT (WINAPI *InitFunc)(void*), void (WINAPI *UninitFunc)()>
struct CCoInitBase
```